doc/development/rfc/python_language_support.md

```diff
@@ -0,0 +1,32 @@
+# RFC 8: Python Language Support
+
+Author of the first draft: Nicklas Larsson
+
+Status: draft
+
+## Introduction
+
+The code base of GRASS GIS consists today (Feb. 2021) of predominantly C code (ca 50 %), Python (ca 30 %) and a smaller amount of C++ (ca 5 %). Each of these languages have evolved significantly in the last 10–20 years.
+There is, however, no clearly stated policy of supported language standard(s), nor mechanism to update this policy when needed or wanted. This result in uncertainty for contributors for what may be allowed and solutions that may not be optimal.
+
+This RFC aims at setting a policy for GRASS GIS project regarding the minimum version support for Python.
+
+## Background
+
+Similarly, the integration of Python within GRASS GIS – driving the GUI and to a large extent replacing shell scripts (bash/sh) for scripting – has gone through a challenging period with the transition from Python 2 to 3. This transition can now be considered completed, but this fact need to be formalised and the minimum version of Python 3 support must be stated.
+
+The minimum support for Python will, in contrast, likely change more often, due to each (minor) version’s 5-years life-time. At this date, the oldest still-alive Python version is 3.6, to be retired in December 2021.
+
+## Version Updates
+
+For a new release of a minor GRASS version, the Python minimum version should be raised if the current minimum Python version reaches end of life or there are any important technical reasons.
```

Comment:
```suggestion
For a new release of a minor GRASS version, the Python minimum version should be raised in the requirements file if the current minimum Python version reaches end of life or there are any important technical reasons.
```

Reply:
I suggest to actually link to (using the relative link):

https://github.com/OSGeo/grass/blob/main/REQUIREMENTS.md#general-requirements

Reply:
> I suggest to actually link to (using the relative link):
> 
> https://github.com/OSGeo/grass/blob/main/REQUIREMENTS.md#general-requirements

done